18F/C2
# [Delivers #96028254] mange user profile

Story: https://www.pivotaltracker.com/story/show/96028254

Expands the `/me` endpoint to allow users to manage their own names. Will keep a header banner present until the first and last name is set in their profiles.

Test:

1. Login and click on your email address in the header nav.
1. Verify your first and last names are set. If not, enter and click *Update profile*
